docs/multicluster/setup-multi-primary.sh

```diff
@@ -0,0 +1,216 @@
+if [ "${ISTIO_VERSION}" == "master" ]; then
+    SAIL_VERSION="latest"
+else
+    SAIL_VERSION="v${ISTIO_VERSION}"
+fi
+
+CTX_CLUSTER1=kind-east
+CTX_CLUSTER2=kind-west
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+if [ ! -d "${CERTS_DIR}" ]; then
+    mkdir -p "${CERTS_DIR}"
+    pushd "${CERTS_DIR}"
+    curl -fsL -o common.mk "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/tools/certs/common.mk"
+    curl -fsL -o Makefile.selfsigned.mk "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/tools/certs/Makefile.selfsigned.mk"
+    make -f Makefile.selfsigned.mk root-ca
+    make -f Makefile.selfsigned.mk east-cacerts
+    make -f Makefile.selfsigned.mk west-cacerts
+    popd
+fi
+
+# 1. Create kind clusters
+
+kind get clusters | grep east || kind create cluster --name east
+kind get clusters | grep west || kind create cluster --name west
+
+# 2. Install sail operator on each cluster:
+
+kubectl config use-context "${CTX_CLUSTER1}"
+kubectl get ns sail-operator --context "${CTX_CLUSTER1}" || make -C "${SCRIPT_DIR}/../.." deploy
+kubectl config use-context "${CTX_CLUSTER2}"
+kubectl get ns sail-operator --context "${CTX_CLUSTER2}" || make -C "${SCRIPT_DIR}/../.." deploy
+
+# 3. Create istio-system namespace on each cluster and configure a common root CA. 
+
+kubectl get ns istio-system --context "${CTX_CLUSTER1}" || kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+kubectl --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+kubectl get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+  --from-file=${CERTS_DIR}/east/ca-cert.pem \
+  --from-file=${CERTS_DIR}/east/ca-key.pem \
+  --from-file=${CERTS_DIR}/east/root-cert.pem \
+  --from-file=${CERTS_DIR}/east/cert-chain.pem
+
+kubectl get ns istio-system --context "${CTX_CLUSTER2}" || kubectl create namespace istio-system --context "${CTX_CLUSTER2}"
+kubectl --context "${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+kubectl get secret -n istio-system --context "${CTX_CLUSTER2}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER2}" \
+  --from-file=${CERTS_DIR}/west/ca-cert.pem \
+  --from-file=${CERTS_DIR}/west/ca-key.pem \
+  --from-file=${CERTS_DIR}/west/root-cert.pem \
+  --from-file=${CERTS_DIR}/west/cert-chain.pem
+
+# 4. Create Sail CR on east
+
+kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: ${SAIL_VERSION}
+  namespace: istio-system
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: east
+      network: network1
+EOF
+
```

Comment:
We should wait for the operator to be ready before attempting to read resource status

```suggestion

kubectl wait --context "${CTX_CLUSTER1}" --for=condition=Available deployment/sail-operator -n sail-operator --timeout=3m
```